cdap-app-templates/cdap-etl/pom.xml

```diff
@@ -0,0 +1,268 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright © 2014 Cask Data, Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <parent>
+    <groupId>co.cask.cdap</groupId>
+    <artifactId>cdap-app-templates</artifactId>
+    <version>3.0.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>cdap-etl</artifactId>
+  <name>CDAP ETL Template</name>
+  <packaging>pom</packaging>
+
+  <modules>
+    <module>cdap-etl-api</module>
+    <module>cdap-etl-core</module>
+    <module>cdap-etl-lib</module>
+    <module>cdap-etl-test</module>
+  </modules>
+
+  <repositories>
+    <repository>
+      <id>continuuity-public</id>
+      <url>https://repository.continuuity.com/content/groups/public</url>
+    </repository>
+  </repositories>
+
+  <properties>
+    <twitter4j.version>4.0.1</twitter4j.version>
+    <mysql.jdbc.version>5.1.6</mysql.jdbc.version>
+  </properties>
+
+  <dependencies>
```

Comment:
Parent pom should only have dependency management, not dependency. Otherwise all these will be dependency of "cdap-etl-api", which is not what we wanted.